### ISSUE TITLE
Replace ':' in series_name and title template variables

### ISSUE THREAD

**Issue by yordanov-alexandar** [crunchy-labs/crunchy-cli#345]
**Describe the bug**
I am using series_name template variable to put the downloaded episode in a folder with series_name name.
When the name contains the character ':' in the file manager I am not seeing the actual name but some gibberish(like this "DCIRWL~N"). When using the command like I can see that the name is ok just the display in the file browser is wrong.
Context: the download happens on a server and I am looking at the files that are shared through samba - that's what is most likely responsible for the problem - when I made the same folder name locally it displayed correctly.
I have only seen this for the series_name variable but I am guessing it would apply for the title also and maybe other variables I am not aware of.

**To Reproduce**
Download any series  that contain ':' in the name with this in the command line
```
 -o "{series_name}/[S{season_number}E{episode_number}] {title}.mkv" 
```
and access it through samba on a linux machine

**Expected behavior**
Folder name displays correctly

**Client (please complete the following information):**
 - OS: Arch Linux for both the server and client - client is with KDE DE
 - Version: 3.2.5

**Additional context**

**Comment by kralverde** [comment]
`--universal-output` is probably what you need

**Comment by yordanov-alexandar** [comment]
What is "--universal-output" and how it's used - I don't see it in the crunchy-cli description

**Comment by kralverde** [comment]
It basically sanitizes the file outputs for windows, which is the most restrictive. In your case it will remove colons from the output file.

**Comment by kralverde** [comment]
Gotcha, my bad

**Comment by yordanov-alexandar** [comment]
Ok thank you. I'll wait for the next release.

**Comment by bytedream** [comment]
The flag is now available in [v3.3.0](https://github.com/crunchy-labs/crunchy-cli/releases/tag/v3.3.0)
